about.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="styles/main.css">
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <header>
+    <h1>AC GILLETTE</h1>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="projects.html">Projects</a>
+      <a href="writing.html">Writing</a>
+    </nav>
+  </header>
+  <body>
+    <section id="about">
+
+    </section>
+  </body>
+  <footer>
+    <p>
+      site belongs to me
```

Comment:
Noticed that this footer is showing up on the left-hand side of the page. Did you want it to be on the left-hand side? and did you follow this same aesthetic choice throughout all your pages?